lerna.json

```diff
@@ -1,6 +1,10 @@
 {
   "packages": [
-    "packages/**"
+    "packages/atomic",
```

Comment:
Will happen locally mostly, but lerna throws an error since we now have the package @coveo/headless twice (even though it's private). The solution is to be more specific in the package array